sound/soc/sof/Makefile

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
-		control.o trace.o utils.o sof-audio.o
+		control.o trace.o utils.o sof-audio.o ipc-stream.o
```

Comment:
Update the commit message with the new name for sof/intel/intel-ipc.c ->  sof/ipc-stream.c

Reply:
Done. Thanks Iulia!